exercises/02_check_palindrom/02_check_palindrom_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import "testing"
+
+func TestIsPalidndrom(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		input string
+		want  bool
+	}{
+		{"empty", "", true},
```

Comment:
Could you add more trivial tests, like "a", "ab"? What about strings with unicode characters?